backend/tests/TeamEventsAPI.test.ts

```diff
@@ -0,0 +1,88 @@
+import { PermissionError } from '../src/errors';
+import { teamEventsCollection } from '../src/firebase';
+import {
+  createTeamEvent,
+  deleteTeamEvent,
+  getAllTeamEvents,
+  updateTeamEvent
+} from '../src/team-eventsAPI';
+import jaggerData from './data/jagger-profile.json';
+
+const adminUser = { email: 'hl738@cornell.edu' } as IdolMember;
+const nonAdminUser = { email: 'aa2235@cornell.edu' } as IdolMember;
```

Comment:
So you should actually be an admin member (I forgot to do that when you joined the team). Probably should change to some other user